apps/publisher/themes/default/js/list_assets.js

```diff
@@ -189,7 +189,10 @@ var createQuery = function(options) {
     var input = $('#inp_searchAsset').val();
     var category = options.category || undefined;
     var searchQueryString = '?';
-    input = setDefaultSearchQuery(input);
+    input = input.trim();
+    if (input.indexOf(":") == -1 && input.trim() !== "") {
```

Comment:
input is already trimmed in line 192. do a null check first.
if (input && input.indexOf(":") == -1)